tests/suites/main_test.function

```diff
@@ -77,6 +79,12 @@ SUITE_POST_DEP
 /*----------------------------------------------------------------------------*/
 /* Test dispatch code */
 
+#if defined(__unix__) || (defined(__APPLE__) && defined(__MACH__))
+#include <unistd.h>
```

Comment:
I think we need this for `dup()`, so I do not have a problem with it. However, perhaps it would be ideal to move it to the other includes in helpers.function. Alternatively, we can create a includes.function (this is not difficult to do).